docs/source/anatomy_credentials.rst

```diff
@@ -35,9 +35,7 @@ a reminder of the purpose. Defaults can also be provided, useful for running on
          "purpose": "Token for accessing a 3rd party API service"
        },
        {
-         "name": "SECRET_THE_THIRD",
-         "purpose": "Usually a big secret but sometimes has a convenient non-secret default, like a sandbox or local database",
```

Comment:
I removed it as the stated purpose for that secret was no longer valid and thought it might be good to show that purpose isn't always required? Maybe we should make purpose required though as it's probably bad practice not to provide one

Reply:
Ah, gotcha. Yes, I think we should make purpose required but we can do that in a separate PR/issue, not for now. In the meantime, reinstate but just rephrase so it doesn't refer to defaults.